config.toml.sample

```diff
@@ -19,36 +19,44 @@ graphql_url = "http://127.0.0.1:3001"
 source = "neutron"
 rpc = "https://rpc-kralum.neutron-1.neutron.org:443/"
 jitter = 10000
-pagination_limit = 10000
+limit = 10000
```

Comment:
Previously, I renamed it from `limit` to `pagination_limit` to make it similar to other sources. What's the reason for renaming it back to `limit`?

Reply:
good catch